R/derive_var_anrind.R

```diff
@@ -5,6 +5,10 @@
 #'   `ANRLO`, `ANRHI`, and `AVAL` are expected and if `use_a1hia1lo` is set to `TRUE`,
 #'   `A1LO` and `A1H1` are expected as well.
 #'
+#' @param signif_dig Number of significant digits to use when comparing values.
+#'
+#'   Significant digits used to avoid floating point discrepancies when comparing numeric values.
```

Comment:
Ohhhh..maybe a quick link to the blog post on this issue?

Reply:
Done, added same link to `derive_var_atoxgr_dir()` and also Lab grading vignette.